app/views/pages/developer-guide/user-uploads/generating-uploading-pdf.liquid

```diff
@@ -106,6 +106,7 @@ query {
   }
 }
 ```
+{% include 'alert/tip', content: 'To change the PDF's orientation consider setting the page size in your css to landscape: @page { size: landscape; }' %}
```

Comment:
Thank you @ScottBReynolds! What would you think about writing it like this:
"To change the PDF's orientation, set the page size..."

Reply:
Sounds good!